src/frontend/src/utils/iiConnection.ts

```diff
@@ -353,7 +353,6 @@ export const creationOptions = (
 ): PublicKeyCredentialCreationOptions => {
   return {
     authenticatorSelection: {
-      userVerification: "preferred",
```

Comment:
Can you briefly explain why you remove the following? 
```
userVerification: "preferred",
attestation: "direct",
```
* It seems `userVerification: "preferred"` is the default, so removing it does not have an effect, correct? Also, is there a reason for not using the seemingly more secure "required"? 
* Removing attestation seems to now default to "none". So this means we are just not interested in attestation here?

Reply:
> It seems userVerification: "preferred" is the default, so removing it does not have an effect, correct? Also, is there a reason for not using the seemingly more secure "required"?

It doesn't have an effect, right. This is what causes users to have to enter the additional PIN for yubikeys on Windows, but there's no setting which actually disables it on creation. I'm just going to revert this change, there's no point.

> Removing attestation seems to now default to "none". So this means we are just not interested in attestation here?

To make `attestation` useful, we'd have to verify it on the other end, which we're not doing. We have some plans for it in the future, but it would be a large undertaking. @Dfinity-Bjoern has some more details if you're curious.
I just figured we shouldn't make the authenticator produce data we don't end up using, and potentially even exclude using security keys that can't produce attestion.

Reply:
Thanks for providing the context. Sounds good to me. 

Regarding "Also, is there a reason for not using the seemingly more secure "required"?" 
I assume we actually don't want to ask the users to enter a pin when they use their Yubikey.